includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -75,8 +75,10 @@ protected function should_show_theme_extra() {
 		return (
 			current_user_can( 'install_themes' ) &&
 			current_user_can( 'switch_themes' ) &&
-			! is_multisite() &&
-			! current_theme_supports( 'woocommerce' )
+			! is_multisite() && (
+				! current_theme_supports( 'woocommerce' ) ||
```

Comment:
This conditional is pretty hard to read now - let's move the theme support + theme checking part into a variable before being used here.